.github/workflows/cuda-gpu-ci.yml

```diff
@@ -41,6 +43,7 @@ jobs:
           conda activate sklearn
           pip install --verbose --no-build-isolation --config-settings editable-verbose=true --editable .
       - name: Run array API tests
+        id: tests
```

Comment:
Q: why is this needed?

If needed, maybe use a more specific id such as "run-array-api-tests"?

Reply:
It is left over from before. Removed